uniffi_bindgen/src/pipeline/general/default.rs

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use super::*;
+
+pub fn pass(module: &mut Module) -> Result<()> {
+    // fields and arguments need `DefaultValue` conversion.
+    module.visit_mut(|arg: &mut Argument| {
+        // eek, this sucks - can't borrow it to clone in the body
+        let ty = arg.ty.clone();
+        arg.visit_mut(|default: &mut DefaultValue| {
```

Comment:
In hindsight, I think my suggestion of a second `visit_mut` was silly.  Can you just use `arg.default` directly and avoid the clone?

Reply:
...and my feeling is that we should be able to avoid these kinds clones in most cases.  Even if `DefaultValue` wasn't so easy to reach, I think you could use `arg.default.visit_mut()` and borrow `arg.ty` inside the closure.

Reply:
doh! Yes, thanks.